pkg/config/server/server.go

```diff
@@ -154,6 +154,9 @@ type ConfigFileRuntime struct {
 	// FlushItemsThreshold is the default number of items to hold in memory until flushing to the database
 	FlushItemsThreshold int `mapstructure:"flushItemsThreshold" json:"flushItemsThreshold,omitempty" default:"100"`
 
+	// FlushStrategy is the strategy to use for flushing the buffer
+	FlushStrategy buffer.BuffStrategy `mapstructure:"flushStrategy" json:"flushStrategy" default:"DYNAMIC"`
```

Comment:
Have we tested that setting `STATIC` as an env var properly marshals in the static buffer strategy?

Reply:
The reason I ask is that I've had problems with viper marshaling config in weird ways before...